.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,24 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea for the nf-core website
```

Comment:
I don't think the text here is right. It should be an idea for the nf-core/dualrnaseq pipeline